.github/workflows/validate-renovate-config.yml

```
@@ -0,0 +1,78 @@
+name: Validate Renovate Configuration
+
+on:
+  # 1. Läuft bei Pull Requests (auch bei Updates)
+  pull_request:
+    paths:
+      - 'Global-Config/**'
+      - '.github/workflows/**'
+  
+  # 2. Läuft bei Push auf main Branch
+  push:
+    branches:
+      - main
```

```suggestion
      - main
      - master
```

Maybe it is usefull to also include the default git branch name, in case the contributer did not changed it to "main" at the installation of git or after, but instead brain-afk the setup.

If so we may need to also adjust all the text, mentions instead of "main" > "main"/"master"

---

GitHub: Since October 2020, new repositories automatically have main set as the default branch.
GitLab: Since the end of 2020, main has also been used as the standard for new projects.
Bitbucket: Bitbucket now also uses main in new repositories.
Azure DevOps: main is increasingly the standard here as well.

We won't be seeing a lot of master branches, I'd assume.